packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -140,7 +140,10 @@ export function callUser({userIdOrEmail, constraints = {audio: true, video: true
       .then((localMediaStream) => {
         const call = spark.phone.dial(
           userIdOrEmail,
-          Object.assign({offerOptions: {offerToReceiveVideo: true, offerToReceiveAudio: true}}, {localMediaStream})
+          Object.assign({
+            constraints,
```

Comment:
Yea that's what we thought, but parts of the rest of the webrtc and calling code expects it for state management. So when it isn't set change events don't fire correctly.